src/Node/Expression/CallExpression.php

```diff
@@ -57,6 +57,10 @@ protected function compileCallable(Compiler $compiler)
 
     protected function compileArguments(Compiler $compiler, $isArray = false): void
     {
+        if ($isArray) {
```

Comment:
```suggestion
        if (func_num_args() >= 2) {
```

Like this so that explicitly passing `false` is also covered?

Reply:
Makes sense